src/main/java/cs2103t/duke/date/MyDate.java

```diff
@@ -0,0 +1,102 @@
+package cs2103t.duke.date;
+
+/**
+ * Encapsulates a date and time.
+ */
+public class MyDate {
```

Comment:
Hi there, can i ask why you decided to create your own MyDate object instead of simply using the LocalDate / LocalTime that is already pre-defined by java? That would save you alot of trouble . Thanks!